### PR TITLE
BCDA-3801 - Add Tealium tracking to static site

### PR DESCRIPTION
**Pull request by mtrang1263**
### Fixes [BCDA-3801](https://jira.cms.gov/browse/BCDA-3801)

Using this [implementation guide](https://confluence.cms.gov/display/BLSTANALYT/CMS.gov+Subdomains), we have added Tealium tracking to all of the pages in the static site.

Deployment job updated to leverage this: https://github.com/CMSgov/bcda-ops/pull/600
### Change Details
1. Updating `default.html` to add Tealium tracking to all of the pages.
2. Use JEKYLL_ENV to use different set of tracking depending if the site is deployed to prod or not. [See](https://jekyllrb.com/docs/configuration/environments/).
3. Update docker-compose file to default the environment to dev. We have a PR on the bcda-ops side to set the variable to prod.

### Security Implications

- [x] new software dependencies
This PR adds Tealium tracking to our static site.
- [ ] security controls or supporting software altered
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications
- [x] no PHI/PII is affected by this change

### Acceptance Validation
1. Verified that we see Tealium events after adding the integration.
Screen Shot 2020-11-17 at 2 51 05 PM

2. Waiting for confirmation from Blast team that we have correct integration points. [See](https://jira.cms.gov/browse/BLSTANALYT-1562).

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,5 +5,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfiles/Dockerfile.static_site
+    environment:
+      - JEKYLL_ENV=dev
```
Comment (msnook) [inline docker-compose.yml:9]: Do we need to update our deployment script to set `JEKYLL_ENV` appropriately?  For example: https://github.com/CMSgov/bcda-ops/blob/master/jenkins_files/Jenkinsfile.deploy_static_site#L71-L72

Comment (msnook) [reply docker-compose.yml:9]: Nevermind!  I just saw this PR with the appropriate change: https://github.com/CMSgov/bcda-ops/pull/600

**Comment by msnook** [comment]
For transparency, should we add Tealium to our [Tech Stack documentation in Confluence](https://confluence.cms.gov/pages/viewpage.action?pageId=120235389)?

**Review by Dave-Sukharan — APPROVED** [review]
No real idea how the script works! But, I've seen a demo of the results and I approve.

**Pull request merged**